### PR TITLE
Split auth Store into seperate files

### PR DESCRIPTION
**Pull request by kenoir**
To stop this file becoming too big

### CODE REVIEW

File: common-lib/src/main/scala/com/gu/mediaservice/lib/BaseStore.scala [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import scala.concurrent.duration._
+
+import org.slf4j.LoggerFactory
+
+import com.amazonaws.auth.AWSCredentials
+import com.amazonaws.AmazonServiceException
+
+import org.apache.commons.io.IOUtils
+
+import _root_.play.api.libs.json._
```
Comment (akash1810) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/BaseStore.scala]: any reason for `_root`?

**Comment by akash1810** [comment]
👍

File: common-lib/src/main/scala/com/gu/mediaservice/lib/BaseStore.scala (hunk 1 above)
Comment (kenoir) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/BaseStore.scala]: There is a `play` package in `common-lib` at the same level so it's necessary to be explicit here.

Comment (akash1810) [inline common-lib/src/main/scala/com/gu/mediaservice/lib/BaseStore.scala]: oh 😞

**Pull request merged**